.redhat-ci.yml

```diff
@@ -1,12 +1,9 @@
 host:
-    distro: fedora/24/cloud
-branches:
-    - master
+    distro: fedora/24/atomic
 packages:
```

Comment:
So basically to make this work you'd change the `tests` line to do something like:

```
cat >/root/tests.sh << EOF
#!/bin/bash
set -xeuo pipefail
yum -y install gcc python3-tox python3-virtualenv
tox
EOF
docker run --rm -v /root:/root -v $(pwd):/code --chdir /code fedora:24 /root/tests.sh
```